.github/copilot-instructions.md

```diff
@@ -0,0 +1,139 @@
+# GitHub Copilot Instructions for autorest.typescript
+
+This repository focuses on the TypeSpec TypeScript generator, which generates TypeScript client libraries from [TypeSpec](https://typespec.io/) specifications. TypeSpec is the recommended approach for describing cloud service APIs and generating client libraries.
+
+> **Note:** While this repository also contains the AutoRest TypeScript generator for OpenAPI/Swagger specifications, we recommend using TypeSpec for new projects. The AutoRest generator is in maintenance mode and will be deprecated in the future.
```

Comment:
Also this might not be very helpful. I Think a note on the projects that are actually in maintenance mode stating, this is in maintenance mode, should ignore unless explicitly asked to use or edit